api/app/controllers/spree/api/address_books_controller.rb

```diff
@@ -2,32 +2,42 @@ module Spree
   module Api
     class AddressBooksController < Spree::Api::BaseController
       # Note: the AddressBook is the resource to think about here, not individual addresses
+      before_filter :load_user_addresses
 
       def show
-        render_address_book
+        authorize! :display_address_book, address_book_user
```

Comment:
Because we're using the user for authorization, could we just: `authorize! :display, address_book_user`? Or is that too permissive?